src/App/components/Header/components/SettingsModal/SettingsModal.tsx

```diff
@@ -144,6 +154,10 @@ const SettingsModal: React.FC = () => {
           <Divider
             className={clsx({ [style.divider]: true, [style.darkDivider]: theme === THEME.dark })}
           />
+          <TimezoneBlock localSettings={localSettings} handleSetSettings={handleSetSettings} />
+          <Divider
```

Comment:
I guess we can create separate component "Divider" that will be in folder "UI" and contain fuctionality with useStyle hook